src/cdk/scrolling/virtual-scroll-viewport.spec.ts

```diff
@@ -827,9 +834,8 @@ function triggerScroll(viewport: CdkVirtualScrollViewport, offset?: number) {
   encapsulation: ViewEncapsulation.None,
 })
 class FixedSizeVirtualScroll {
-  @ViewChild(CdkVirtualScrollViewport) viewport: CdkVirtualScrollViewport;
-  @ViewChild(CdkVirtualForOf) virtualForOf: CdkVirtualForOf<any>;
-  @ViewChild(CdkVirtualForOf, {read: ViewContainerRef}) virtualForViewContainer: ViewContainerRef;
+  @ViewChild(CdkVirtualScrollViewport, {static: true}) viewport: CdkVirtualScrollViewport;
+  @ViewChild(CdkVirtualForOf, {static: true}) virtualForOf: CdkVirtualForOf<any>;
```

Comment:
instead of casting everywhere how about doing it once here:

```ts
... virtualForOf: CdkVirtualForOf<any> & { /* add private methods to type */
  _detachView: ...,
  _createEmbeddedVeiwAt: ...,
}
```

Reply:
Oh, good call. Can change 👍

Reply:
Hmm I can't seem to get that to work. TS complains that the methods are inaccessible.